volatility3/framework/plugins/linux/graphics/fbdev.py

```diff
@@ -255,22 +255,28 @@ def _generator(self):
             vollog.error(
                 "PIL (pillow) module is required to use this plugin. Please install it manually or through pyproject.toml."
             )
-            return None
+            return
 
         kernel_name = self.config["kernel"]
         kernel = self.context.modules[kernel_name]
 
         if not kernel.has_symbol("num_registered_fb"):
-            raise exceptions.SymbolError(
-                "num_registered_fb",
-                kernel.symbol_table_name,
-                "The provided symbol does not exist in the symbol table. This means you are either analyzing an unsupported kernel version or that your symbol table is corrupt.",
+            vollog.error(
+                '"num_registered_fb" symbol does not exist in the symbol table. This means you are either analyzing an unsupported kernel version,  your symbol table is corrupt, or the fbdev driver is compiled as a kernel module.'
+            )
+            return
+
+        try:
+            num_registered_fb = kernel.object_from_symbol("num_registered_fb")
+        except exceptions.SymbolError:
+            vollog.error(
```

Comment:
Do we know the exact lowest required version number of dwarf2json?  I think we've got code that can check and warn on such things? (see [here](https://github.com/volatilityfoundation/volatility3/blob/d13cf7ae2205584ec89dbc5088d0c5d38fc9acbc/volatility3/framework/plugins/linux/kmsg.py#L563) )

Reply:
hmm the thing there is that, if the fbdev driver is compiled into the kernel, then the plugin will work with the old symbols tables. The `object_from_symbol` call only fails when both conditions are met (compiled as module + old symbol table).

Reply:
Ok, so then the check belongs in here?  The point is we should be able to tell whether their symbol table is too old or if that's not the problem?